scripts/log-capture

```diff
@@ -60,14 +60,21 @@ _to_log ip a
 _to_log ip r
 _to_log journalctl
 
+if [[ -e /sys/firmware/efi ]]; then
+    _to_log ls -l /sys/firmware/efi/efivars
+fi
+
 cp /etc/resolv.conf ${OUTDIR}
-cp /tmp/*.log ${OUTDIR}
-cp /root/lorax-packages.log ${OUTDIR}
+cp /tmp/*.log ${OUTDIR} 2>/dev/null
```

Comment:
How would that fail? It's plain copying.

Reply:
I was testing in a Fedora container which has some differences to the anaconda install environment.  The big one being those logs don't exist in the container.

Reply:
I reworked this to use `stat` to check for existence